### ISSUE TITLE
"We couldn't load the extension from" error

### ISSUE THREAD

**Issue by ylcet** [rcmaehl/MSEdgeRedirect#272]
```
Issue #1:
When opening MS Edge using the shortcut I get a popup error: "We couldn't load the extension from: C:\Users\John\AppData\Local\Microsoft\Edge\User. Manifest file is missing or unreadable"
```

```
Issue #2:
When clicking on any link from the Windows 11 news widget near the Windows Start button, Chrome opens up and shows:
"Redirect Notice
 The previous page is sending you to https://www.cbsnews.com/news/fbi-hunt-somali-pirate-leader-fatal-kidnapping-4-americans/.

 If you do not want to visit that page, you can [return to the previous page](https://www.google.com/url?q=https://www.cbsnews.com/news/fbi-hunt-somali-pirate-leader-fatal-kidnapping-4-americans/#)."

and the URL shown in the address bar is: "https://www.google.com/url?q=https://www.cbsnews.com/news/fbi-hunt-somali-pirate-leader-fatal-kidnapping-4-americans/"

```

Chrome is currently set as my default browser. I ran the MSEdgeRedirect-Cleanup Tool.ps1 tool to uninstall and try again with the current beta release but same thing happens. 

I've uninstalled it now using the powershell cleanup script again but when I check the MSEdge About page it shows:

"Version 114.0.1823.51 (Official build) (64-bit)
An error occurred while checking for updates: We couldn't create that component (error code 1: [0x80004005](https://go.microsoft.com/fwlink/?linkid=2068169&0x80004005) -- system level)."

This error may have existed before so this bug report isn't about this error.

**Comment by rcmaehl** [comment]
Issue 1 (extension error):

I'll probably need the log files for this, or a list of extensions (if any) you have installed in Edge

Issue 2 (redirect notice):

Switch MSN News to DuckDuckGo. It'll operate pretty much the same but you won't get that error

Issue 3 (updating):

Resolved in 0.7.4.0

**Comment by rcmaehl** [comment]
Never received reply from reporter
